provisioning/roles/nginx/templates/nginx.conf.j2

```diff
@@ -0,0 +1,32 @@
+user  nginx;
+worker_processes  {{ worker_processes }};
+
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+
+
+events {
+    worker_connections  {{ max_clients }};
+}
+
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  /var/log/nginx/access.log  main;
+
+    sendfile        on;
+    #tcp_nopush     on;
```

Comment:
Tu écris un autre playbook `rolling_upgrades.yml` ou quelque chose comme ça qui devra se charger d'orchestrer ta mise à jour.

Souvent, on fait:

- on désactive le monitoring de la machine (Nagios et co)
- on retire la machine d'un HAProxy ou load balancer

- on la met à jour (git pull, build the project in production, blablabla)

Souvent dans cette mise à jour, tu vas notifier des choses comme "restart NGINX", "restart Gunicorn"

Et du coup Ansible va les queue dans une liste de "trucs à faire à la fin"
Une fois le playbook fini, il exécute cette queue.
(c.f. http://docs.ansible.com/ansible/guide_rolling_upgrade.html)